intentkit/clients/cdp.py

```diff
@@ -10,6 +9,7 @@
 )
 
 from intentkit.abstracts.skill import SkillStoreABC
+from intentkit.config.config import config
```

Comment:
don't import config from skill, you can only use config by self._skill_store.get_system_config